src/test/resources/basic-type-test/Request/CustomBasicType.elm

```diff
@@ -0,0 +1,24 @@
+{-
+  This file was generated by endpoints-elm interpreter.
+  Do not edit this file manually.
+
+  See https://github.com/scalalandio/endpoints-elm for more information.
+-}
+
+module Request.CustomBasicType exposing (..)
+
+import Http
+import HttpBuilder exposing (RequestBuilder)
+import Json.Decode as Decode
+import Bool.Extra
+import Maybe.Extra
+
+
+
+
+localdateechoGet : LocalDate -> LocalDate -> RequestBuilder TimeOnly
+localdateechoGet  local-date =
```

Comment:
`local-date` is not legal elm variable name; either we name it `localDate` or normalize this name in code generation